website/docs/guides/migration/versions/04-upgrading-to-dbt-utils-v1.0.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Migrating to dbt utils 1.0"
+---
+
+# dbt utils 1.0 Migration Guide
+
+For the first time, dbt utils is crossing the major version boundary. From [last month’s blog post](https://www.getdbt.com/blog/announcing-dbt-v1.3-and-utils/): 
+
+> It’s time to formalise what was already unofficial policy: you can rely on dbt utils in the same way as you do dbt Core, with stable interfaces and consistent and intuitive naming.
```

Comment:
@joellabes it looks like the quote is different from what's in the blog - do we want to change the below so it matches with the blog?

```suggestion
>at’sIt’s time for dbt utils to tick over to 1.0 and commit to what was already unofficial policy: you can rely on dbt utils in the same way as you do dbt Core, with stable interfaces and consistent and intuitive naming.
```